tests/unittests/CoreGraphics/CGContextTests.mm

```diff
@@ -285,7 +285,6 @@ void cgContextPathCompare(NSArray* expected, NSArray* result) {
 
     static ComPtr<ID2D1RenderTarget> renderTarget;
     static void SetUpTestCase() {
-        // TODO GH#1124: When CGBitmapContext lands, we don't need to do this manually.
```

Comment:
This can be replaced with CGBitmapContextCreateImage.
Actually reverting this file and sending out a PR to fix this.

---
In reply to: [108798952](https://github.com/Microsoft/WinObjC/pull/2364#discussion_r108798952) [](ancestors = 108798952)